pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/AggregationResultHolder.java

```diff
@@ -30,6 +30,12 @@ public interface AggregationResultHolder {
    */
   void setValue(double value);
 
+  /**
+   * Set the 'primitive long' aggregation result.
```

Comment:
(minor) same for other places
```suggestion
   * Set the 'primitive int' aggregation result.
```